wandb/sdk/launch/runner/sagemaker_runner.py

```diff
@@ -128,6 +168,13 @@ def run(
 
         # Create a sagemaker client to launch the job.
         sagemaker_client = session.client("sagemaker")
+        log_client = None
+        try:
+            log_client = session.client("logs")
+        except Exception:
+            wandb.termwarn(
+                "Failed to connect to cloudwatch logs, logs will not be available"
+            )
```

Comment:
because we are catching literally all exceptions, maybe also print the error w/o stacktrace? it might be possible to mess up the session/client object and it not be a problem with logs at all